packages/@lwc/template-compiler/src/parser/utils/html-element-attributes.ts

```diff
@@ -71,11 +72,9 @@ const HTML_ELEMENT_ATTRIBUTE_MAP = {
         'rel',
         'rev',
         'shape',
-        'tabindex',
         'target',
         'type',
     ],
-    abbr: ['title'],
```

Comment:
Some of these mappings were dropped, but again, this should only affect warnings.